contrib/lkt/extensions/mains/lkt_toolbox.adb

```diff
@@ -41,6 +41,13 @@ procedure Lkt_Toolbox is
          Long   => "--check-only",
          Help   => "Only output the errors");
 
+      package Flag_Invalid is new Parse_Flag
+        (Parser => Parser,
+         Short  => "-I",
+         Long   => "--flag-invalid-nodes",
+         Help   => "Flag nodes that generate errors that are not annotated"
+                   & " with the @invalid annotation. Also flag nodes "
```

Comment:
For consistency and easy check of missing spaces, could you move the the trailing space to the beginning of next line?

```
& "... Also flag nodes"
& " annotated with …"
```

Also given that `@invalid` is for decls and that we check both decls with and without `@invalid`, I would suggest to call this option `--check-invalid-decls`.